src/os/bluestore/StupidAllocator.cc

```diff
@@ -119,7 +118,7 @@ int StupidAllocator::allocate(
   // search up (from origin, and skip searched extents by hint)
   for (bin = orig_bin; bin < (int)free.size(); ++bin) {
     p = free[bin].begin();
-    end = hint ? free[bin].lower_bound(hint) : free[bin].end();
+    auto end = hint ? free[bin].lower_bound(hint) : free[bin].end();
```

Comment:
can you squash this change into the previous one? as we are not expecting the fix-the-fix commit in a single PR.